### PR TITLE
[WIP] Update python-cdk branch to CDK v1.9.0

### PR DESCRIPTION
**Pull request by dstroppa**
*Issue #, if available:*

*Description of changes:* Updates to CDK modules and instructions after run through.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: module-2/README.md [hunk 1]
```diff
@@ -274,7 +274,7 @@ export class EcrStack extends cdk.Stack {
 }
 ```
 
-**Note:** We are assigning the instance of our `ecr.Repository` to a readonly property so that it may be referenced by other stacks.
+> **Note:** We are assigning the instance of our `ecr.Repository` to a readonly property so that it may be referenced by other stacks.
```
[the remark this answers is not in the document]
Comment (dstroppa) [reply module-2/README.md]: It's a blockquote, makes the note stand out more

File: module-6/cdk/lib/xray-stack.ts [hunk 2]
```diff
@@ -60,19 +60,19 @@ export class XRayStack extends cdk.Stack {
         displayName: 'MythicalMysfitsQuestionsTopic',
         topicName: 'MythicalMysfitsQuestionsTopic'
     });
-    topic.addSubscription(new subs.EmailSubscription("REPLACE@EMAIL_ADDRESS"));
+    topic.addSubscription(new subs.EmailSubscription("moviolone@gmail.com"));
```
Comment (DavidChristiansen) [inline module-6/cdk/lib/xray-stack.ts]: Please check that this was intentional

File: module-7/cdk/lib/xray-stack.ts [hunk 3]
```diff
@@ -60,19 +60,19 @@ export class XRayStack extends cdk.Stack {
         displayName: 'MythicalMysfitsQuestionsTopic',
         topicName: 'MythicalMysfitsQuestionsTopic'
     });
-    topic.addSubscription(new subs.EmailSubscription("REPLACE@EMAIL_ADDRESS"));
+    topic.addSubscription(new subs.EmailSubscription("moviolone@gmail.com"));
```
Comment (DavidChristiansen) [inline module-7/cdk/lib/xray-stack.ts]: Please check that this was intentional

Comment (DavidChristiansen) [inline module-7/cdk/lib/xray-stack.ts]: Please check that this was intentional

File: module-6/cdk/lib/xray-stack.ts (hunk 2 above)
Comment (DavidChristiansen) [inline module-6/cdk/lib/xray-stack.ts]: Please check that this was intentional

**Pull request merged**